EIPS/Increase block rewards to 5 ETH.md

```diff
@@ -0,0 +1,34 @@
+---
+eip: <to be assigned>
+title: Increase block rewards to 5 ETH
+author: Ben Tinner (@Terra854)
+discussions-to: https://www.reddit.com/r/ethereum/comments/k4252u/thoughts_about_increasing_block_rewards_till_we/
```

Comment:
Please use a forum like http://ethereum-magicians.com instead of Reddit. It is a more suitable long-term destination for discussion (see #2967).